internal/backends/sqlite/metadata/registry.go

```diff
@@ -127,6 +127,28 @@ func (r *Registry) DatabaseList(ctx context.Context) []string {
 	return r.p.List(ctx)
 }
 
+// DatabaseSize returns the size of the database in bytes.
+//
+// If database does not exist, 0 is returned.
+func (r *Registry) DatabaseSize(ctx context.Context, dbName string) (int64, error) {
```

Comment:
An alternative approach is to make `DatabaseList` return both names and sizes, but the current approach looks clearer to me.